### DISCUSSION TITLE
How can I change the minimum length and width, the default length and width, and whether or not to hide the default title bar of the Window of my Uno Platform program and it will take effect on Windows, MacOS and Linux?

### DISCUSSION THREAD

**Discussion by Rwagsu** [unoplatform/uno#16644] [Q&A] [answered]
I'm making an application using Uno Platform 5.2, but now I want to change the default window size and minimum window size of MainWindow and remove the title bar, I want these changes to take effect in Windows, MacOS and Linux, but I don't know how to do all this, I tried my best to find out a lot of things, but these I've tried my best to find a lot of things but none of them can help me, so does Uno Platform offer anything to help me do this? If not, how can I do this?

**Comment by MartinZikmund** [comment, marked as answer]
Most of these features will be added soon in [AppWindow sizing and positioning](https://github.com/unoplatform/uno/pull/16535) PR I'm working on. For minimum window size there is currently no equivalent API in WinUI, so we will probably add a Uno-specific solution for now.

For title bar customization, we will need further work to achieve this, as it is highly platform specific. What specific customizations are most important to you currently?

**Reply by Rwagsu** [reply]
Thanks for the replies, very much looking forward to what you guys come up with!
I think the features I need at the moment are more customization of the window, comparing Uno to Avalonia, Uno's window is really missing a lot of things, I think the window is a very visible element in a desktop platform, it probably determines how much of an eye-catching look a program gives, and a lot of people should want to customize it, the features I want at the moment are the ability to adjust the window's length and width, the window's minimum width and hide the title bar, and the position of the window in the screen, I think some of these needs should be needed, such as some people with OCD (like me, haha 😉)
However, I'd also like to ask in passing, for some of the MacOS stuff (like shortcut commands, etc.), do you guys have support for them? Or any plans to support them? I'm probably going to switch to working on macOS afterward, and I'm more interested in this question, so I hope you guys can help me out, thanks!
I may be slow to respond to your messages because I've had a cold lately, sorry about that.
